src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC, useRef } from "react";
+import DropDownArrow from "../icons/DropDownArrow";
+import styles from "./Dropdown.module.css";
+
+interface Props {
+  options: Array<string>;
+  whenSelected: () => void;
+}
+
+const Dropdown: FC<Props> = ({ options, whenSelected }: Props) => {
```

Comment:
legge til ...rest dersom man har behov for det :)

Reply:
doone